pkg/mount/mount_test.go

```diff
@@ -96,6 +98,14 @@ func mountTest(t *testing.T, source, dest string) {
 	require.NoError(t, err, "Failed in unmount")
 }
 
+func forceUnmountTest(t *testing.T, source, dest string) {
```

Comment:
There should also be others tests to invoke Unmount twice so that your code path gets actually executed. 
- Invoke Unmount twice, second should fail when options are not set
- Invoke Unmount twice, second should succeed when options are set